packages/mgt-chat/src/statefulClient/Caching/MessageCache.ts

```diff
@@ -11,6 +11,7 @@ interface CachedMessageData extends CacheItem, GraphCollection<ChatMessage> {
 
 export class MessageCache {
   private get cache(): CacheStore<CachedMessageData> {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
This suppression should not be necessary.

There are a lot of introduced suppressions in this PR that seem to be unnecessary. Sometimes VSCode does not do well with newly introduced code until a full build has been performed and the ESLint server restarted.